src/Exception/OidcInvalidCodeChallengeMethodException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Drenso\OidcBundle\Exception;
+
+use LogicException;
+
+class OidcInvalidCodeChallengeMethodException extends LogicException
```

Comment:
This isn't a logic exception, as it currently used to indicate that the IdP doesn't support the requested challenge method right?